update doc for release 0.6.0 (#100)
openelb/website: content/en/docs/getting-started/usage/openelb-ip-address-assignment.md

  metadata:
    name: nginx
    namespace: project
    annotations:
      lb.kubesphere.io/v1alpha1: openelb
  spec:
    selector:
      app: nginx
    type: LoadBalancer
    ports:
      - name: http
        port: 80
        targetPort: 80
    externalTrafficPolicy: Cluster
  ```

### There is a default eip in the cluster

Here is an example to illustrate. As shown in the figure: 

* The `default-eip` is set as the default eip using the annotation `"eip.openelb.kubesphere.io/is-default-eip": "true"`. 
* Namespace `namespace-1` and `namespace-2` have the label `label: test`, which is used by the `eip-selector`'s `namespaceSelector` to match and select. 
* The `eip-ns` EIP has a namespaces field that specifies `namespace-1`. This allows `eip-ns` to match and be allocated to `namespace-1`. 
* The `eip` EIP does not have any special configuration. It can only be used for specified allocation and cannot be automatically allocated.

![ipam-bind-to-namespace-1](/images/en/docs/getting-started/usage/ip-address-assignment/ipam-bind-to-namespace-1.svg)

| Namespace | Available EIP Count |
|--|--|
| `namespace-1` | 3 |
| `namespace-2` | 2 |
| `namespace-3` | 1 |

  {{< notice note >}}

>   When the number of available EIPs is greater than 1, there will be some priority selection to choose the final EIP to use. 

Reviewer: ```suggestion
  When the number of available EIPs is greater than 1, the final EIP will be chosen according to the priorities. 
```
